tests/integration/components/file-editor-column-test.js

```diff
@@ -2,54 +2,81 @@ import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
 moduleForComponent('file-editor-column', 'Integration | Component | file editor column', {
-  integration: true,
-  beforeEach() {
-    this.addColumnCalled = false;
-    this.removeColumnCalled = false;
-    this.showFileTreeCalled = false;
-
-    this.set('col', '2');
-
-    this.on('addColumn', () => { this.addColumnCalled = true; });
-    this.on('removeColumn', () => { this.removeColumnCalled = true; });
-    this.on('showFileTree', () => { this.showFileTreeCalled = true; });
-
-    this.render(hbs`{{file-editor-column col=col
-                                         file=null
-                                         allFiles=null
-                                         keyMap=null
-                                         numColumns=2
-                                         fileTreeShown=false
-                                         contentsChanged="contentsChanged"
-                                         addColumn=(action "addColumn")
-                                         removeColumn=(action "removeColumn")
-                                         showFileTree=(action "showFileTree")
-                                         }}`);
-  }
+  integration: true
 });
 
 test('it calls addColumn when the add column glyph is clicked', function(assert) {
   assert.expect(1);
 
-  this.$('.glyphicon-plus').click();
+  this.set('externalAction', () => {
+    assert.ok(true, 'addColumn action was called');
+  });
+
+  this.render(hbs`
+    {{file-editor-column col='2' numColumns=2 addColumn=(action externalAction)}}
+  `);
 
-  assert.ok(this.addColumnCalled, 'addColumn was called');
+  this.$('.glyphicon-plus').click();
 });
 
 test('it calls removeColumn when the remove column glyph is clicked', function(assert) {
   assert.expect(1);
 
-  this.$('.glyphicon-remove').click();
+  this.set('externalAction', () => {
+    assert.ok(true, 'removeColumn action was called');
+  });
+
+  this.render(hbs`
+    {{file-editor-column col='2' removeColumn=(action externalAction)}}
+  `);
 
-  assert.ok(this.removeColumnCalled, 'removeColumn was called');
+  this.$('.glyphicon-remove').click();
 });
 
 test('it calls showFileTree when the show file tree glyph is clicked', function(assert) {
   assert.expect(1);
 
-  this.set('col', '1');
+  this.set('externalAction', () => {
+    assert.ok(true, 'showFileTree action was called');
+  });
+
+  this.render(hbs`
+    {{file-editor-column col='1' fileTreeShown=false showFileTree=(action externalAction)}}
+  `);
 
   this.$('.glyphicon-chevron-right').click();
+});
+
+test('it calls contentChanged with true when changing the content via the code editor', function(assert) {
+  assert.expect(1);
+  const enterKeyEvent = { keyCode: 3 };
+
+  this.set('externalAction', (isUserChange) => {
+    assert.ok(isUserChange, 'contentChanged action was called');
```

Comment:
I'm on mobile now so I could miss something, but what do you mean? The description is correct(imo) on the main test method above, and this assertion would say that it expected a true value (if it would fail). Did you mean to add that it expects a true value in this assertion?